src/main.ts

```diff
@@ -89,6 +92,12 @@ export class Minifier {
 
     // given a base path, preserve file directory structure
     var subFilePath = filePath.replace(this._minifierOptions.basePath, '');
```

Comment:
Line 91 makes it absolute: `this._minifierOptions.basePath = path.resolve(process.cwd(), this._minifierOptions.basePath);`